Client/Application/AppDelegate.swift

```diff
@@ -256,23 +256,28 @@ extension AppDelegate: UINavigationControllerDelegate {
 
 extension AppDelegate {
     private func configureCrashReporter() {
-        let mainBundle = NSBundle.mainBundle()
-        let breakpad = BreakpadController.sharedInstance()
+        BreakpadController.sharedInstance().start(true)
+
+        // Add in custom crash-stats keys
+        if profile?.prefs.boolForKey("SendCrashReports") ?? false {
```

Comment:
We probably don't want to prompt in the case that it doesn't exist since we will prompt the user on app launch everytime in that case